src/components/Image/Infinite/ImagesCard.tsx

```diff
@@ -67,8 +71,42 @@ export function ImagesCard({ data: image, height }: { data: ImagesInfiniteModel;
                   <ImageGuard.Content>
                     {({ safe }) => (
                       <>
-                        {!isBlocked && <ImageGuard.Report context="image" position="top-right" />}
-                        <ImageGuard.ToggleImage position="top-left" />
+                        <Group
+                          position="apart"
+                          align="start"
+                          spacing={4}
+                          sx={{
+                            position: 'absolute',
+                            top: 0,
+                            left: 0,
+                            width: '100%',
+                            zIndex: 10,
+                            padding: theme.spacing.sm,
+                          }}
```

Comment:
Since SX is unsupported from mantine 7, and this is a style anyway, let's use the plain old style :D. 1 less thing to change